### DISCUSSION TITLE
登录不上啊，

### DISCUSSION THREAD

**Discussion by benabcben** [ZuodaoTech/everyone-can-use-english#856] [Q&A] [answered]
![CleanShot 2024-07-20 at 21 20 06@2x](https://github.com/user-attachments/assets/efa1b4f6-b758-41bb-a6b8-0e1b47703155)
![Uploading 1721481656202.jpg…]()

**Comment by czyrichard** [comment]
我这边也是

**Reply by 3158565951** [reply]
me too

**Comment by benabcben** [comment, marked as answer]
我看几个小时前才发了新版本，咋回事啊

czyrichard ***@***.***> 于2024年7月20日周六 22:42写道：

**Comment by an-lee** [comment]
解决办法： https://github.com/ZuodaoTech/everyone-can-use-english/discussions/853#discussioncomment-10101808
